content/en/Getting started/Pentest objectives/stack.md

```diff
@@ -19,7 +19,7 @@ technologies can vary by asset:
 When you create or update a Web, Mobile, API, or combined [asset](/getting-started/assets/), you can [add a technology stack](/platform-deep-dive/assets/risk-advisories/#add-a-technology-stack-for-your-asset) for it. Technologies that you selected from a predefined list on the asset details page populate in the **Technology Stack** field in the pentest wizard.
 
 - You can add more technologies for your pentest, in addition to those that you specified for your asset earlier.
-- We don't show a predefined list of technologies in the **Technology Stack** list in the pentest wizard, as opposed to the asset details page. Type a technology, and press Enter to confirm.
+- We don't show a predefined list of technologies in the **Technology Stack** list in the pentest wizard, as opposed to the asset details page. Type a technology, and press <kbd>Enter</kbd> to confirm.
```

Comment:
@mjang-cobalt Experimenting with the <kbd> tag in several files.

Reply:
I love your willingness to experiment. However, I haven't seen this kind of graphic with keyboard entries before. I'd rather not set up a new design pattern in our docs. 

OTOH, if you can show me other software docs that use this graphic for keyboard entries -- of if you have the support of our designers, then let's put this into our style guide.

<img width="986" alt="Screenshot 2023-01-05 at 6 29 23 AM" src="https://user-images.githubusercontent.com/84352037/210803517-49830833-9770-4926-8e5c-d33372cfb3e2.png">

Reply:
@mjang-cobalt I saw this recommendation in the Google Developer Style Guide: https://developers.google.com/style/ui-elements#keyboard

I agree that it doesn't look perfect when rendered, so I'm removing this tag for the time being. I'll keep this in mind for our (future) brainstorming sessions about the portal look and feel.